app/assets/stylesheets/application.scss

```diff
@@ -2,3 +2,14 @@
 *= require foundation_and_overrides
 *= require font-awesome
 */
+nav {
```

Comment:
This should be in its own file that is required within this file. This file should be reserved for requiring other stylesheets